### DISCUSSION TITLE
Which branch does username.github.io point to for documentation?

### DISCUSSION THREAD

**Discussion by Sayam753** [community/community#23815] [Other Feature Feedback, Questions, & Ideas]
<p>Hello all<br>
I am using mkdocs to build static files. I wish to use <code>mkdocs gh-deploy -b</code> option to build static files in a separate branch.<br>
Since,  I have setup a repo named .github.io according to GitHub pages guide, now I am wondering at which branch should I store my static files? Either <code>master</code> or <code>gh-pages</code>.<br>
In other words, which branch does &lt;<a href="http://username.github.io" rel="nofollow noopener">username.github.io</a>&gt; point to for hosting static content?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You can refer to the <a href="https://help.github.com/en/github/working-with-github-pages/about-github-pages#publishing-sources-for-github-pages-sites">help documentation</a> for the full details, but user and organization pages must publish from the <code>master</code> branch.</p>
